Positionspapier v0.5: Kapitelstruktur komplett, LSR-Feedback + Versionsmerge (#26)
oeme-github/INA-ePA-und-Patientenportale: positionspapier.md

---

## 2. Einleitung

### 2.1 Ziele des Arbeitskreises

*[Platzhalter]*

### 2.2 Begriffsklärungen

*[Platzhalter]*

---

## 3. Methodisches Vorgehen

### 3.1 Grundprinzipien

Der AK Patientenportale hat sich auf einen bewusst prozessgetriebenen Ansatz geeinigt. Ausgangspunkt ist die Überzeugung, dass eine tragfähige Standardisierung des Datenaustauschs im Gesundheitswesen nicht von Systemen ausgehen kann — sie muss vom Prozess her gedacht werden.

Der gewählte Ansatz folgt konsequent dieser Reihenfolge:

> **Erst die Prinzipien — dann der Prozess — dann die Daten.**

Die Systemebene wird zunächst ausgeklammert und erst in einem zweiten Schritt als Mapping auf Prozesse und Daten betrachtet. Das bedeutet nicht, dass bestehende technische Lösungen ignoriert werden. Im Gegenteil: Die Ist-Analyse benennt explizit, was heute bereits existiert und funktioniert — VSDM, ISiK, ePA-Infrastruktur, FHIR-Profile, IHE-Profile. Dieser Bestand ist der Ausgangspunkt. Der Prozessrahmen gibt den Maßstab vor, an dem bestehende Lösungen gemessen und Lücken identifiziert werden.

Daraus leiten sich vier Grundprinzipien ab:

**Prozess vor Daten vor System.**

> Systeme unterstützen Prozesse. Prozesse werden von Akteuren ausgeführt und erzeugen, verändern oder löschen Daten. Die Systemebene ist Mittel zum Zweck — nicht Ausgangspunkt.

Reviewer: ```suggestion
Systeme unterstützen Prozesse. Prozesse werden von Akteuren ausgeführt und erzeugen, verändern oder löschen Datenobjekte. Die Systemebene ist Mittel zum Zweck — nicht Ausgangspunkt.
```

*Prozess vor Daten [...]*: *Datenobjekte* sind Prozess-inhärent, der Term *Daten* ist an der Stelle nicht gemeint (meiner Auffassung nach).

**Keine führenden Systeme.**

> Es gibt keine primären oder sekundären Systeme. Daten gehören dem Prozess — und letztlich dem Patienten. Systemgrenzen sind kein strukturgebendes Element.

Reviewer: ```suggestion
Es gibt keine primären oder sekundären Systeme. Datenobjekte gehören dem Prozess — und letztlich dem Patienten. Systemgrenzen sind kein strukturgebendes Element.
```
Vereinheitlichung auf *Datenobjekt*

**Datenobjekte vor Datenflüsse.**

> Im Fokus stehen nicht Schnittstellen zwischen Systemen, sondern Datenobjekte: Was wird wann von wem erzeugt, verändert, bereitgestellt oder gelöscht?

Reviewer: ```suggestion
Im Fokus stehen nicht Schnittstellen zwischen Systemen, sondern Datenobjekte: Was wird wann von wem erzeugt, verändert oder gelöscht?
```

Konsistenz mit der in 3.2 (Z. 74) abschließend definierten Operation {E, V, G}. „bereitgestellt" ist eine Fluss-/Nutzungsoperation, kein Lebenszyklusschritt — und steht im Widerspruch zur Überschrift dieses Prinzips.

> **Strukturierte Daten vor Dokumenten.**

Reviewer: ```suggestion
**Strukturierte Datenobjekte vor dokumentenbasierter Darstellung.**
```
Mit *Dokument* ist hier meiner Meinung nach die Repräsentationsform gemeint. Anhand von Datenobjekten, die einer definierten Struktur folgen, lassen sich bspw. Dokumente generieren. Wir wollen ja erreichen, dass nicht PDFs gespeichert werden, sondern die in den PDFs gespeicherten Inhalte bedeutungserhaltend gespeichert und ausgetauscht werden können.

> Datenobjekte sollen maschinenlesbar und strukturiert sein. Die menschenlesbare Darstellung ist Aufgabe der Systeme — ob auf dem Bildschirm, im Ausdruck oder in anderer Form. Strukturierte Datenobjekte ermöglichen Automatisierung, Entscheidungsunterstützung und sektorenübergreifende Weiterverarbeitung. Sie sind das Ziel — nicht die Ausnahme.

Reviewer: ```suggestion
Datenobjekte sollen maschinenlesbar und strukturiert sein. Ein Dokument (z. B. ein PDF) ist dabei selbst ein Datenobjekt in unstrukturierter, menschenlesbarer Repräsentation — kein Gegensatz zum Datenobjekt, sondern dessen Darstellungsform. Die menschenlesbare Darstellung ist Aufgabe der Systeme — ob auf dem Bildschirm, im Ausdruck oder in anderer Form. Strukturierte Datenobjekte ermöglichen Automatisierung, Entscheidungsunterstützung und sektorenübergreifende Weiterverarbeitung. Sie sind das Ziel — nicht die Ausnahme.
```

Klarstellung, dass ein Dokument selbst ein Datenobjekt (in unstrukturierter Repräsentation) ist und kein Gegenpol dazu.

Kein Element kann ohne die anderen beiden gedacht werden. Diese Formel ist der gemeinsame Anker der Gruppe.

---

### 3.2 Prozessmodell

Der Patientenpfad wird in drei temporale Phasen gegliedert:

| Phase | Beschreibung |
|---|---|
| **Vor dem Krankenhaus** | Alles, was zeitlich vor dem stationären oder ambulanten Aufenthalt stattfindet — von der Terminanfrage bis zur Aufnahmevorbereitung |
| **Im Krankenhaus** | Der Aufenthalt selbst, von der Aufnahme bis zur Entlassung — einschließlich Pflege, Diagnostik, Behandlung und Entlassungsplanung |
| **Nach dem Krankenhaus** | Nachsorge und Weiterversorgung — von der Bereitstellung der Entlassdokumente bis zum Abschluss des Behandlungsfalls |

Innerhalb dieser Phasen wurden **25 Prozessschritte** modelliert. Sie repräsentieren die aus Sicht der Arbeitsgruppe zentralen Schritte im Patientenpfad — ohne Anspruch auf Vollständigkeit. Jeder Prozessschritt wird durch drei Dimensionen beschrieben:

- **Akteur** — Wer führt den Schritt aus? (Details siehe Kap. 3.3)
- **Datenobjekt** — Was entsteht, verändert sich oder wird gelöscht?
- **Operation** — Wird das Datenobjekt erzeugt (E), verändert (V) oder gelöscht (G)?

> Diese Modellierung erlaubt eine systemunabhängige Beschreibung des Patientenpfads. Eine Übersicht der im Rahmen der Unterarbeitsgruppen analysierten Prozessschritte ist in Kapitel 4.3 dargestellt.

Reviewer: ```suggestion
Die Operation ist dabei das Verb des Prozessschritts: Sie konkretisiert, was im Sinne der Leitformel (Kap. 3.1) tatsächlich am Datenobjekt geschieht. Diese Modellierung erlaubt eine systemunabhängige Beschreibung des Patientenpfads. Eine Übersicht der im Rahmen der Unterarbeitsgruppen analysierten Prozessschritte ist in Kapitel 4.3 dargestellt.
```

Abstrakt gesprochen: Akteure führen Operationen auf Datenobjekten aus, was als die beschreibende Tätigkeit / Aktivität des Prozessschrittes aufzufassen ist.

Neben den explizit modellierten Prozessschritten gibt es **prozessbegleitende Elemente**, die an mehreren Stellen des Patientenpfads auftreten und nicht einem einzelnen Schritt zugeordnet werden können. Dazu gehört die **Patientenberatung und -aufklärung**: Aufklärungsgespräche, Informationsmaterialien und evidenzbasierte Entscheidungshilfen begleiten den Patienten von der Terminvorbereitung über die Einwilligungserteilung bis zur Entlassung. Die dabei entstehenden und eingesetzten Datenobjekte — Aufklärungsbögen, Gesundheitsinformationen, Entscheidungshilfen — sind im Prozessmodell noch nicht explizit abgebildet. Dies ist ein offener Punkt, der einer eigenständigen Modellierung bedarf (vgl. Kap. 4.4, Querschnittsthema).

---

> ### 3.3 Daten-, Informationsdomänen und Akteure

Reviewer: ```suggestion
### 3.3 Datenobjekte, Informationsdomänen, Datenräume und Akteure
```

#### Datenobjekte

> Prozessschritte stehen in Bezug zu Datenobjekten — sie erzeugen, verändern, löschen oder nutzen sie. Manche Schritte bringen Datenobjekte hervor, andere stellen sie lediglich bereit oder zeigen sie an. Datenobjekte sind die zentralen Informationseinheiten des Modells. Beispiele: Terminanfrage, Überweisung, Anamnese, Diagnose, Medikationsplan, Pflegedokumentation, Arztbrief.

Reviewer: ```suggestion
Prozessschritte stehen in Bezug zu Datenobjekten: Manche erzeugen, verändern oder löschen sie (Lebenszyklus), andere stellen sie lediglich bereit, nutzen oder zeigen sie an, ohne den Lebenszyklus zu verändern. Datenobjekte sind die zentralen Informationseinheiten des Modells. Beispiele: Terminanfrage, Überweisung, Anamnese, Diagnose, Medikationsplan, Pflegedokumentation, Arztbrief.
```

Lebenszyklus (E/V/G) und reine Bereitstellung/Anzeige/Nutzung in derselben Aufzählung vermischen Objekt- und Flussebene. Trennung ohne Inhaltsverlust.

---

### 3.4 Entwicklung eines interaktiven Prozessschaubilds

Parallel zur inhaltlichen Arbeit hat die Arbeitsgruppe ein **interaktives Prozessschaubild** entwickelt, das alle 25 Prozessschritte mit ihren Attributen (Phase, Akteure, Datenobjekte, Informationsdomäne, Datenräume, Rechtsgrundlagen, Ist-Analyse) visualisiert und filterbar macht.

Das Tool ist als statische HTML-Anwendung ohne externe Abhängigkeiten realisiert und steht allen AK-Mitgliedern über GitHub Pages zur Verfügung. Die Datenpflege erfolgt über einen integrierten Editor, der Änderungen direkt in das Datenmodell zurückschreiben kann.

Das Schaubild erfüllt zwei Funktionen: Es ist Arbeitswerkzeug für die Gruppe (Ist-Analyse, Lückenidentifikation) und kommunikatives Hilfsmittel für die Präsentation der Ergebnisse nach außen.

---

## 4. Problemstellung und Herausforderungen

### 4.1 Systemrollen und Verantwortlichkeiten

*[Platzhalter — Beschreibung der Rollen von ePA, KIS und Patientenportal gemäß Agenda-Entwurf]*

### 4.2 Internationale Perspektive

*[Platzhalter — EHDS, Portale und ePAs in anderen Ländern, z. B. Dänemark, Estland, Niederlande]*

### 4.3 Prozessüberblick

> Die Arbeitsgruppe hat im Rahmen von drei Unterarbeitsgruppen (UAG) — je eine für jede Phase des Patientenpfads — 12 der 25 Prozessschritte im Detail analysiert. Diese Schritte wurden ausgewählt, weil sie einen unmittelbaren Bezug zu den in Kapitel 5 formulierten Handlungsempfehlungen haben. Eine vollständige Analyse aller 25 Prozessschritte war im verfügbaren Zeitrahmen nicht umsetzbar. Die methodische Grundlage ist jedoch gelegt: Alle weiteren Prozessschritte lassen sich nach demselben Verfahren — Ist-Analyse, Lückenidentifikation, Forderungsableitung — schrittweise aufarbeiten.

Reviewer: Die Argumentationskette stimmt hier meiner Meinung nach nicht: Wir haben erst Forderungen (Handlungsempfehlungen) formuliert und sie dann Prozessschritten zugeordnet (so steht es hier sinngemäß).

Die vorgestellten Schritte wurden doch sicher aufgrund von *Relevanz, Priorität oder damit verbundenen Problemen* aus Hersteller-Perspektive (allg. Stakeholder-Perspektive) gewählt und anhand dessen die Empfehlungen in den UAGs abgeleitet? Vielleicht nochmal abstimmen, was genau die Auswahlkriterien waren, um dies auch nachvollziehbar und plausibel zu argumentieren.

| 6 | Einwilligungen erteilen | Patient | Einwilligungserklärung / Widerspruch | Consent Management / QES |

#### Phase 2: Im Krankenhaus

| Nr. | Prozessschritt | Akteur(e) | Datenobjekt | Forderungsbezug |
|---|---|---|---|---|
| 7 | Digitaler Check-in | Patient, Verwaltung | Check-in Bestätigung | ISiK-Schnittstellen / Portalintegration |
| 8 | Administrative Aufnahme | Verwaltung | Patientenfall / Fallakte | ISiK-Schnittstellen / ePA-Portal-Integration |
| 13 | Bereitstellung ausgewählter Informationen im Portal | Krankenhausarzt, Verwaltung | Laborwerte / Befunde im Portal | Strukturierte Daten / Kritische Informationen |

#### Phase 3: Nach dem Krankenhaus

| Nr. | Prozessschritt | Akteur(e) | Datenobjekt | Forderungsbezug |
|---|---|---|---|---|
| 19 | Bereitstellung Entlassdokumente | Krankenhausarzt, Verwaltung | Arztbrief / Entlassbericht | Portal-Differenzierung (Patienten- vs. Zuweiserportal) |
| 20 | Bereitstellung Medikationsplan | Krankenhausarzt | Medikationsplan | Medikation: verbindlicher Referenzdatensatz |
| 21 | Bereitstellung Dokumente für ePA | Arzt, Verwaltung | Strukturierte Datenobjekte in der ePA | Konsequente ePA-Befüllung / EEHRxF |

> ### 4.4 Prozesse mit TI-Beteiligung — Ist-Analyse und Use Cases

Reviewer: Hier ggf. einheitlich mit Zwischenüberschriften oder Hervorhebungen arbeiten, um Struktur zu vereinheitlichen und somit Navigierbarkeit/Lesbarkeit zu verbessern: Status Quo, Lücke, Use Case, Forderung

→ Forderungen: Consent Management; QES im Portal (EUDI-Wallet / GesundheitsID) (→ Kap. 5, kurzfristig/langfristig)

---

#### Schritt 7: Digitaler Check-in

Krankenhäuser bieten Terminbuchungsfunktionen im Portal an. Mit ISiK Terminplanung, HL7 FHIR Appointment und HL7 SIU existieren Standards. Einzelne KIS-Hersteller bieten FHIR-Schnittstellen; in einigen Häusern ist die Integration bereits umgesetzt.

Viele KIS-Hersteller bieten FHIR-Schnittstellen für Terminbuchung noch nicht oder nur gegen hohen Aufpreis an. Eine durchgängige, standardisierte Integration zwischen Portal und KIS ist die Ausnahme.

**Use Case:** Ein Patient checkt über das Portal ein und bestätigt seine Anwesenheit digital. Das Portal zeigt eine Bestätigungsseite — aber das KIS weiß davon nichts. Die Verwaltung erfasst die Ankunft manuell am Schalter, weil die Systeme nicht integriert sind. Der digitale Check-in ist aus Sicht des Patienten vorhanden, hat aber keinen Effekt auf den Verwaltungsprozess.

→ Forderungen: ISiK Terminplanung als verbindlicher Standard durchsetzen; FHIR-Integration als Vergabekriterium (→ Kap. 5, kurzfristig)

---

> #### Schritt 8: Administrative Aufnahme

Reviewer: Redundant oder ergänzend zu Zeile 266 *Schritt 6: Einwilligung erteilen*?

Einwilligungsdokumente sind nicht standardisiert. Drittsysteme verursachen hohe Kosten. Die erzeugten Dokumente sind meist PDFs — keine strukturierten, maschinenlesbaren Daten. Aufklärungsbögen und Informationsmaterialien, die im Kontext der Einwilligungserteilung eingesetzt werden, liegen nicht als strukturierte Datenobjekte vor — weder in der ePA verfügbar noch für Folgesysteme zugänglich.

**Use Case:** Bei der Aufnahme unterschreibt ein Patient einen Behandlungsvertrag auf einem Tablet. Das Drittsystem des Krankenhauses speichert das Ergebnis als PDF im KIS. Das nächste Krankenhaus nutzt ein anderes System — neue Formulare, neue Unterschriften. Die strukturierten Informationen aus der Einwilligung (welche Eingriffe, welche Bedingungen) stehen keinem Folgesystem zur Verfügung.

→ Forderungen: Einheitlicher technischer Standard für Einwilligungsdokumente; QES als Zielstandard (→ Kap. 5, kurzfristig/mittelfristig)

---

> #### Schritt 13: Bereitstellung ausgewählter Informationen im Portal

Reviewer: Hier werden doch zwei Probleme zusammen betrachtet: die doppelte Datenbereitstellung sowie die automatisierte Bereitstellung kritischer Informationen - ggf. auftrennen?

---

#### Schritt 20: Bereitstellung Medikationsplan

Medikationsplan und elektronische Medikationsliste (eML) existieren in der ePA und als Ausdruck. Standards sind vorhanden (KBV Medikationsplan, FHIR MedicationStatement). Das KIS führt eine eigene Medikationsliste.

Es fehlt eine Harmonisierung über Systeme und Sektoren hinweg. Übertragungswege, die eine automatisierte Verarbeitung ermöglichen, sind nicht durchgängig etabliert. Welche Version des Medikationsplans die aktuell gültige ist, bleibt unklar.

**Use Case:** Ein Patient verlässt das Krankenhaus mit einem aktualisierten Medikationsplan. Der Hausarzt hat seine eigene Version aus dem letzten Quartal. Die Apotheke hat eine dritte, die ePA eine vierte. Alle vier Versionen weichen voneinander ab. Niemand hat die Autorität — und kein System hat die Logik — eine davon als verbindlich zu erklären. Das Risiko von Wechselwirkungen und Dosierungsfehlern ist real.

→ Forderungen: Medikation als verbindlicher Referenzdatensatz mit klarer Autorenschaft; europaweit einheitliche strukturierte Medikationsliste (→ Kap. 5, kurzfristig/mittelfristig)

---

#### Schritt 21: Bereitstellung Dokumente für ePA

> Die ePA 3.0 ist technisch verfügbar. Spezifikationen für Dokumente, Medikationsliste und der gesetzliche Rahmen sind vorhanden. FHIR und IHE-Standards sind definiert.

Reviewer: Wieder gewisse Redundanzen zu Schritt 6 und 13 - Consent Management und Gesundheitsinformationen/belastende Situationen/ Ethik?

Diese Maßnahmen sind ohne grundlegende strukturelle Änderungen umsetzbar und adressieren unmittelbare Hürden im heutigen Betrieb.

- **Frühzeitige Identifikation über Gesundheits-ID ermöglichen** — einschließlich Unterstützung des EUDI-Wallet für eine sichere, einrichtungsübergreifende Anmeldung im Portal.

- **Schneller TI-Zugriff für Patientenportale** — Einbau eines PoPP-Moduls (Proof of Patient Presence) in das Portal, um frühzeitigen Zugriff auf TI-Fachdienste (insbesondere ePA, aber auch eRezept und VSDM) zu ermöglichen.

- **ISiK-Schnittstellen konsequent umsetzen** — ISiK ist als Mindeststandard bereits gesetzlich verankert. Die Umsetzung in der Fläche fehlt: Definierte Schnittstellen auf Basis von ISiK für den modularen Einsatz von Krankenhausportalen — schnellere Einführung, weniger Aufwand, herstellerunabhängig und austauschbar.

- **Konsequente ePA-Befüllung** — Alle für die medizinische Versorgung relevanten Informationen sollen in die ePA eingestellt werden — bevorzugt in strukturierter Form; wo strukturierte Formate noch fehlen, übergangsweise als PDF. Besonderes Augenmerk: Medikation. Es darf nur einen verbindlichen, aktuellen Referenzdatensatz geben.

> - **Keine Sekundärnutzung und keine proprietäre Bindung** — Patientendaten, die über Krankenhausportale verarbeitet werden, sind keine Handelsware. Das Krankenhaus bleibt als DSGVO-Verantwortlicher für ihre ordnungsgemäße Verarbeitung zuständig — die Hoheit liegt beim Patienten. Daraus folgt zweierlei: Portal-Anbieter dürfen Patientendaten nicht für eigene Zwecke (Analysen, Benchmarking, kommerzielle Weiterverarbeitung) nutzen — entsprechende Klauseln in Verträgen sind nicht akzeptabel. Portaldaten dürfen zudem nicht in proprietären Systemen eingeschlossen werden — Datenportabilität und Interoperabilität sind Mindestanforderungen an jeden Anbieter.

Reviewer: Besser *Keine kommerzielle Sekundärnutzung* benennen, sonst ggf. Verwechselung mit *Sekundärdatennutzung* i.S.v. GDNG/ Forschungszwecke?
